question-servers/elements/pl_answer_panel.py

```diff
@@ -18,5 +18,8 @@ def parse(element_html, element_index, data):
 def grade(element_html, element_index, data):
     return data
 
-def testcase(element_html, element_index, data):
+def test(element_html, element_index, data):
     return data
+
+def file(element_html, element_index, data):
```

Comment:
Matt and I went back and forth about this. As you know, server.py works the way you describe. What are your thoughts?